instrumentation/introspector/_swagger-helper/main.go

```diff
@@ -60,7 +60,7 @@ func main() {
 	}
 	sw.writeString(")\n")
 
-	err = os.Rename(tmpF.Name(), outFileName)
+	err = MoveFile(tmpF.Name(), outFileName)
```

Comment:
os.Rename gives error when used in docker with volume:

go run _swagger-helper/main.go -in=./introproto
{"level":"fatal","loginstance":"global_default","writeDuration":"36.10µs","time":"2020-05-22T13:18:01.498270331Z","caller":"instrumentation/introspector/_swagger-helper/main.go:65","message":"failed move file from /tmp/swghelper_590706880.go to swagger_const_gen.go: rename /tmp/swghelper_590706880.go swagger_const_gen.go: invalid cross-device link"}
exit status 1
##[error]instrumentation/introspector/swagger.go:8: running "go": exit status 1

So just copy a file